src/wasm.h

```diff
@@ -1428,6 +1428,7 @@ class Module {
   std::vector<std::unique_ptr<Import>> imports;
   std::vector<std::unique_ptr<Export>> exports;
   std::vector<std::unique_ptr<Function>> functions;
+  std::vector<std::unique_ptr<FunctionType>> externTypes;
```

Comment:
What is an "extern type"? The 4 things above it are actual properties of modules. Is this something for internal use? It seems like it should have a different API and not be alongside those 4. But, I'm not sure yet I even see the need for it - why not just use a function type?